istioctl/cmd/add-to-mesh.go

```diff
@@ -316,6 +316,10 @@ func injectSideCarIntoDeployment(client kubernetes.Interface, deps []appsv1.Depl
 	for _, dep := range deps {
 		log.Debugf("updating deployment %s.%s with Istio sidecar injected",
 			dep.Name, dep.Namespace)
+		if dep.Name == istioEgressgateway || dep.Name == istioIngressgateway {
+			fmt.Fprintf(writer, "failed to inject sidecar to deployment resource %s.%s. Skipping.\n", dep.Name, dep.Namespace)
```

Comment:
I think using `multierror.Append` will be consistent here ?
https://github.com/istio/istio/blob/e1e8825bc02a7aab683a37c02790d6bc3a3a0a9c/istioctl/cmd/add-to-mesh.go#L325-L326